### DISCUSSION TITLE
Make slider anchors visible when hovering over

### DISCUSSION THREAD

**Discussion by Lyti4n** [ppy/osu#29287] [Ideas]
### Type

Game behaviour

### Bug description

Currently in lazer you have to click on a slider to see and edit its slider anchors. this makes everything take way longer since you have to click on it each time. I honesty dont see any reason for it other than it looks cleaner I guess? It's not worth making it awful to use.

### Version

2024.731.0

### Logs

[compressed-logs.zip](https://github.com/user-attachments/files/16485640/compressed-logs.zip)

**Comment by peppy** [comment]
There are cases where control points can be overlapping other things on screen you're trying to interact with. Not sure about this one.

**Comment by Lyti4n** [comment]
i've warmed up to how this works right now, it definitely still slows things down but i think the cleanliness it creates is probably worth it
